### PR TITLE
[VEN-2836]: Bootstrap liquidity for Optimism markets

### PR DESCRIPTION
**Pull request by GitGuru7**
## Description

Resolves VEN

### CODE REVIEW

**Review by chechu — APPROVED** [review]

**Review by kkirka — APPROVED** [review]
LGTM

File: simulations/vip-373/bscmainnet.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { expect } from "chai";
+import { BigNumber, Contract } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { expectEvents } from "../../src/utils";
+import { forking, testVip } from "../../src/vip-framework";
+import vip373, { COMMUNITY_WALLET, TOKEN_REDEEMER, TREASURY, VUSDC, VUSDC_AMOUNT } from "../../vips/vip-373/bscmainnet";
+import ERC20_ABI from "./abi/ERC20.json";
+import VTreasurey_ABI from "./abi/VTreasury.json";
```
Comment (kkirka) [inline simulations/vip-373/bscmainnet.ts:10]:
```suggestion
import VTreasury_ABI from "./abi/VTreasury.json";
```

File: simulations/vip-373/bscmainnet.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    },
+  });
+
+  describe("Post-VIP behavior", async () => {
+    it("should decrease vtreasury balance by the amount used to transfer vtoken corresponding to 25000 usdc", async () => {
+      expect(await vusdc.balanceOf(TREASURY)).to.closeTo(prevTreasuryBalance.sub(VUSDC_AMOUNT), parseUnits("500", 8));
+    });
+
+    it("Should increase USDC balance of the community wallet", async () => {
+      const currUSDcBal = await usdc.balanceOf(COMMUNITY_WALLET);
```
[the remark this answers is not in the document]
Comment (chechu) [reply simulations/vip-373/bscmainnet.ts]: a562d67a4e7a1741908187908ce9dd0e2a244e8f

File: simulations/vip-373/bscmainnet.ts (hunk 1 above)
Comment (chechu) [reply simulations/vip-373/bscmainnet.ts]: a562d67a4e7a1741908187908ce9dd0e2a244e8f

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/373?chainId=56

**Review by chechu — APPROVED** [review]

**Pull request merged**